Solution-Week1/script_7_print_indexvalue.py

```diff
@@ -0,0 +1,14 @@
+#Write A Script To Print Index Value Instead of Printing Vowels In String
+def print_indexval(string):
+    vowels=['a','e','i','o','u','A','E','I','O','U']
+    cpstr=[]

```

Comment:
why you take empty list?

Reply:
Here, I take empty list because first i check all the character and if character is vowel then i replace those character with index value and after then i append those character into empty list.